go/vt/sqlparser/sql.y

```diff
@@ -513,59 +517,127 @@ load_statement:
   {
     $$ = &Load{}
   }
+query_expression_parens:
+  openb query_expression_parens closeb
+  {
+  	$$ = $2
+  }
+| openb query_expression closeb
+  {
+     $$ = $2
+  }
+| openb query_expression locking_clause closeb
+  {
+    setLockInSelect($2, $3)
+    $$ = $2
+  }
 
-select_statement:
-  base_select order_by_opt limit_opt lock_opt into_option
+// TODO; (Manan, Ritwiz) : Use this in create, insert statements
```

Comment:
should this comment be addressed before merging the PR?

Reply:
This PR is already too big, so I don't think we want to do it as part of this PR, but we should do it later. Basically, right now we use the `select_statement` grammar rule in create table or insert into type queries, but `select_statement` also includes selects with into clause which is wrong and should not be passed. So right now we are parsing more in create and insert than we should. We added the TODO to keep track of this, but we will do this change when we also look at the parsing of the create and insert statements and match MySQL's behaviour